src/Misc/FiniteField.jl

```diff
@@ -12,11 +12,6 @@ function FlintFiniteField(p::fmpz; cached::Bool = true)
   return k, k(1)
```

Comment:
Can't `FlintFiniteField(p::fmpz, ...` also go? Flint has that now, too, with signature `FlintFiniteField(char::fmpz, deg::Int, s::Union{AbstractString,Symbol} = :o; cached = true)`.

This leaves the method accepting any `Integer`. For that it'd be great it also had argument `s::Union{AbstractString,Symbol}="o"` to match its "sibling methods".

Reply:
No, Nemo does not have them as far as I can see:
```
julia> FiniteField(2)
ERROR: MethodError: no method matching FlintFiniteField(::Int64)
Closest candidates are:
  FlintFiniteField(::Int64, ::Int64) at ~/.julia/dev/Nemo/src/flint/fq_nmod.jl:664
  FlintFiniteField(::Int64, ::Int64, ::Union{AbstractString, Symbol}; cached) at ~/.julia/dev/Nemo/src/flint/fq_nmod.jl:664
  FlintFiniteField(::Union{fmpz_mod_poly, gfp_fmpz_poly}) at ~/.julia/dev/Nemo/src/flint/fq.jl:633
```

Reply:
My mistake, I somehow completely missed the omitted degree argument

Reply:
Hrmmm.... the "Mark as resolved" button does not show up for this conversation. GitHub bug?